ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1659,6 +1659,9 @@
     color: $theme.notificationErrorColor;
     font-size: .87em;
 }
+.match-details .variants-data a:visited {
```

Comment:
This is not the right place to fix the issue. It is actually caused by a selector that is too broad in some unrelated code: https://github.com/phenotips/phenotips/blob/master/components/gene-panels/ui/src/main/resources/PhenoTips/PanelsDisplay.xml#L552 . That selector can affect much more than just the UCSC links for variants. The problem should be fixed there directly.